### DISCUSSION TITLE
Github is unable to locate my zip file

### DISCUSSION THREAD

**Discussion by Nawi98** [montudor/action-zip#32] [Help wanted]
Hello @montudor , 

I am currently using your github action to zip my file, but unfortunately I am facing this issue : 

![image](https://github.com/montudor/action-zip/assets/87074904/9337f710-dbd1-4ec4-9733-dea6a3cde5a9)

Here is my workflow : 

    `on: workflow_dispatch 
      #schedule:
      # - cron: '0 7 * * 5' # every Friday at 7:00 AM
    jobs:
      run_manual : # run my test
        runs-on: ubuntu-latest
        steps:
          - name: run the Testim s tests with Testim s gitHub action
            #id: tests_ok_ko
            uses: instinctstudios/testim-cli-gh-action@v0.6.0
            continue-on-error: true
            with:
          token: ${{ secrets.TESTIM_TOKEN }} 
          project: ${{ secrets.PROJECT_ID }} 
          grid: Testim-Grid
          suite: to_test_github_integration

      
                
            - name: upload the artifact generated by Testim itself
              uses: actions/upload-artifact@v3
              with:
                name: test_report
                path: /home/runner/work/launch_testim_tests/launch_testim_tests/testim-report.xml
                if-no-files-found: warn
            
            #- name: Find the file
              #run: find / -type f -name testim-report.xml -print
              #run: pwd
      
           
              
            
            - name: parse the file
              run: cat /home/runner/work/launch_testim_tests/launch_testim_tests/testim-report.xml | grep -i -C 1 "<failure" | tee final-report.xml
      
            - name: Zip file
              uses: montudor/action-zip@v1
              with:
                args: zip -qq -r testim-report.zip . -i /home/runner/work/launch_testim_tests/launch_testim_tests/testim-report.xml
      
            #- name: Find the file
            # run: find ${{ github.workspace }} -name 'testim-report.zip' -print
      
            - name: upload the artifact generated by Testim itself
              uses: actions/upload-artifact@v3
              with:
                name: test_report
                path: /home/runner/work/launch_testim_tests/launch_testim_tests/testim-report.zip
                if-no-files-found: warn
            
            - name: Download the zip file 
              uses: actions/download-artifact@v3
              with:
                name: /home/runner/work/launch_testim_tests/launch_testim_tests/testim-report.zip
      `
Regards.

**Comment by montudor** [comment]
It appears the below action is being provided the wrong path. This is likely because the zip action will run in the working directory (`/home/runner` by default if I recall). This means the path is more likely `/home/runner/testim-report.zip` - so this is the file you need to upload.

I would recommend using the `working-directory:` along with relative paths rather than using `/home/runner/absolute/path`.
